### PR TITLE
chore: fix some xref issues

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/page-visibility/pull/52.html" title="Last updated on Aug 1, 2019, 11:21 AM UTC (7bf3711)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/page-visibility/52/24b6bd5...7bf3711.html" title="Last updated on Aug 1, 2019, 11:21 AM UTC (7bf3711)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,7 +173,7 @@ <h2>
       </dl>
       <p>
         The <a>visibility states</a> are reflected in the API via the
-        <a>VisibilityState</a> enum.
+        {{VisibilityState}} enum.
```
Comment (yoavweiss) [inline index.html:176]: Do you have a guide on how one differs from the other?

**Review by yoavweiss — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:176]: you mean `[=Thing=]` and `{{Thing}}`? Full guide coming soon... but it's  [=Thing=] is a concept and {{Thing}} is IDL (same micro-syntax that BikeShed uses).

Comment (yoavweiss) [reply index.html:176]: and what's `<a>Thing</a>` compared to that?

Comment (marcoscaceres) [reply index.html:176]: so, the concept of a "URL" vs the {{URL}} IDL interface, for instance.

Comment (marcoscaceres) [reply index.html:176]: You can search for terms via this https://respec.org/xref/ :)

Comment (marcoscaceres) [reply index.html:176]: That shows you if something is concept, an event, an exception, etc. and how to cite it... and which spec it comes from.

Comment (yoavweiss) [reply index.html:176]: Cool, thanks! :)

Comment (marcoscaceres) [reply index.html:176]: Note, you can now access the xref UI directly from within the spec via the ReSpec pill:

Screenshot_2019-08-01_21_29_50

Comment (marcoscaceres) [reply index.html:176]: Sorry, didn't answer your question exactly: 

> and what's `<a>Thing</a>` compared to that?

that's just an alternative (old) syntax... backwards compatible, but less clear.

**Comment by yoavweiss** [comment]
Thanks for tidying up! :)

**Pull request merged**